tasks/utilities/data_handling/task_fetch_srr_metadata.wdl

```diff
@@ -0,0 +1,52 @@
+version 1.0
+
+task fetch_srr_metadata {
+  input {
+    String sample_accession 
+    String docker = "us-docker.pkg.dev/general-theiagen/biocontainers/fastq-dl:2.0.4--pyhdfd78af_0"
+    Int disk_size = 10
+    Int cpu = 2
+    Int memory = 8
+  }
+  meta {
+    volatile: true
+  }
+
+  command <<< 
+    mkdir -p metadata_output
+    date -u | tee DATE
+
+    # Debug output to show the sample being processed
+    echo "Fetching metadata for sample accession: ${sample_accession}"
+
+    # Use fastq-dl to fetch metadata only
+    fastq-dl --accession ~{sample_accession} --provider SRA --outdir metadata_output --only-download-metadata --verbose
```

Comment:
why limit to provider SRA?

Reply:
I removed provider. I was worried about ENA having different format for output tsv but it appears SRR is still first column